pkg/controller/association/conf.go

```diff
@@ -109,6 +110,7 @@ func ElasticsearchAuthSettings(c k8s.Client, association commonv1.Association) (
 
 // AllowVersion returns true if the given resourceVersion is lower or equal to the associations' versions.
 // For example: Kibana in version 7.8.0 cannot be deployed if its Elasticsearch association reports version 7.7.0.
+// A difference in the patch version is ignored: Kibana 7.8.1 can be deployed alongside Elasticsearch 7.8.0.
```

Comment:
```suggestion
// A difference in the patch version is ignored: Kibana 7.8.1+ can be deployed alongside Elasticsearch 7.8.0.
```

Just to show it's not only +-1 patch.